assets/prefabs/Pineapple/PineappleSeed.prefab

```diff
@@ -0,0 +1,31 @@
+{
+  "parent": "engine:iconItem",
+  "DisplayName": {
+    "name": "Pineapple Seed"
+  },
+  "Item": {
+    "icon": "SimpleFarming:SimpleFarming#pineappleseed",
```

Comment:
Like the pineapple this shouldn't be a mapping to an image atlas - you've got individual images to refer to instead of an atlas.